workflow.md

```diff
@@ -112,9 +112,8 @@ release must be described in the corresponding section.
 
 Each section contains 3 subsections:
```

Comment:
Oh, I see. But how is `Features` different from `Improvements`? Isn't a _feature_ actually an _improvement_? Sound like I'm getting philosophical, but I always prefer to have things as simple as possible.